src/pages/security/_lazy-load.js

```diff
@@ -2,4 +2,5 @@ import Loadable from '@loadable/component'
 
 export const AboutDeriv = Loadable(() => import('./_about-deriv'))
```

Comment:
You probably don't want About Deriv section lazy loadable since it's probably a visible section on load on desktops.